### ISSUE TITLE
DatabaseUpdateListener not compatible with migrate command

### ISSUE THREAD

**Issue by rabauss** [m-vo/contao-facebook-import#25]
The `Controller::reload();` in `DatabaseUpdateListener` is not compatible with the `contao:migrate` command of contao 4.9 - causes: This exception has no message.
https://github.com/m-vo/contao-facebook-import/blob/d2b80a56f2eeb89d56640eaa361a7aab9e3f0271/src/EventListener/DatabaseUpdateListener.php#L64

P. S. the foreign keys are a good idea, but they often cause big problems in the installer!

**Comment by m-vo** [comment]
Yeah that was an ugly workaround anyways. I'm fine with dropping the migrations in a new version.

> P. S. the foreign keys are a good idea, but they often cause big problems in the installer!

see https://github.com/contao/contao/issues/1507

I'm not going to remove foreign key relations from any of my projects. Not having them is the bug. :smile:
